README.md

```diff
@@ -2,11 +2,13 @@
 
 A Windows network throughput benchmark tool.
 
-See Releases section for binaries.
+## Building
 
-# Building
+[![Build](https://github.com/microsoft/ntttcp/actions/workflows/build.yml/badge.svg)](https://github.com/microsoft/ntttcp/actions/workflows/build.yml)
```

Comment:
I actually don't know if it makes sense to put the build badge in this section, since this section is about locally building. I think it would make more sense to put it at the top.